src/components/ProfileMatch/index.tsx

```diff
@@ -39,7 +39,7 @@ const timeCommitmentMatch: MatchField<CaseListing> = {
       ? `Your time commitment (${profileData.hours_per_month} hours/month) ${
           match ? 'meets' : 'does not meet'
         } the minimum for this case (${data.hours_per_week * 4} hours/month).`
-      : 'No information available about Time Commitment',
+      : 'No information available about time commitment',
```

Comment:
i liked it uppercase since it matched the way it's displayed in the case highlights. do you think it's better lower case ?

Reply:
cc @varortz

Reply:
from ad hoc worksesh: lowercase to match Figma